Disable tests which are incompatible with our version of autoconf/autotest

diff -r cfea8989bb9d tests/check-structs.at
--- a/tests/check-structs.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/check-structs.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,11 +1,11 @@ AT_BANNER([struct alignment checker unit
 AT_BANNER([struct alignment checker unit tests])
 
-m4_define([check_structs], [$top_srcdir/build-aux/check-structs])
-m4_define([RUN_STRUCT_CHECKER], 
-  [AT_SKIP_IF([test $HAVE_PYTHON = no])
-   AT_DATA([test.h], [$1
-])
-   AT_CHECK_UNQUOTED([$PYTHON check_structs test.h], [$2], [$3], [$4])])
+dnl m4_define([check_structs], [$top_srcdir/build-aux/check-structs])
+dnl m4_define([RUN_STRUCT_CHECKER], 
+dnl   [dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl    AT_DATA([test.h], [$1
+dnl ])
+dnl    AT_CHECK_UNQUOTED([$PYTHON check_structs test.h], [$2], [$3], [$4])])
 
 AT_SETUP([check struct tail padding])
 RUN_STRUCT_CHECKER(
diff -r cfea8989bb9d tests/classifier.at
--- a/tests/classifier.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/classifier.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,16 +1,16 @@ AT_BANNER([flow classifier unit tests])
 AT_BANNER([flow classifier unit tests])
-m4_foreach(
-  [testname],
-  [[empty],
-   [destroy-null],
-   [single-rule],
-   [rule-replacement],
-   [two-rules-in-one-bucket],
-   [two-rules-in-one-table],
-   [two-rules-in-different-tables],
-   [many-rules-in-one-bucket],
-   [many-rules-in-one-table],
-   [many-rules-in-different-tables]],
-  [AT_SETUP([flow classifier - m4_bpatsubst(testname, [-], [ ])])
-   AT_CHECK([test-classifier testname], [0], [], [])
-   AT_CLEANUP])])
+dnl m4_foreach(
+dnl   [testname],
+dnl   [[empty],
+dnl    [destroy-null],
+dnl    [single-rule],
+dnl    [rule-replacement],
+dnl    [two-rules-in-one-bucket],
+dnl    [two-rules-in-one-table],
+dnl    [two-rules-in-different-tables],
+dnl    [many-rules-in-one-bucket],
+dnl    [many-rules-in-one-table],
+dnl    [many-rules-in-different-tables]],
+dnl   [AT_SETUP([flow classifier - m4_bpatsubst(testname, [-], [ ])])
+dnl    AT_CHECK([test-classifier testname], [0], [], [])
+dnl    AT_CLEANUP])])
diff -r cfea8989bb9d tests/daemon-py.at
--- a/tests/daemon-py.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/daemon-py.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,9 +1,9 @@ AT_BANNER([daemon unit tests - Python])
 AT_BANNER([daemon unit tests - Python])
 
 AT_SETUP([daemon - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([expected])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([expected])
 # Start the daemon and wait for the pidfile to get created
 # and that its contents are the correct pid.
 AT_CHECK([$PYTHON $srcdir/test-daemon.py --pidfile-name=$PWD/pid& echo $! > expected], [0], [ignore], [ignore])
@@ -19,11 +19,11 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --monitor - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([parent])
-AT_CAPTURE_FILE([parentpid])
-AT_CAPTURE_FILE([newpid])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([parent])
+dnl AT_CAPTURE_FILE([parentpid])
+dnl AT_CAPTURE_FILE([newpid])
 # Start the daemon and wait for the pidfile to get created.
 AT_CHECK([$PYTHON $srcdir/test-daemon.py --pidfile-name=$PWD/pid --monitor& echo $! > parent], [0], [ignore], [ignore])
 OVS_WAIT_UNTIL([test -s pid], [kill `cat parent`])
@@ -62,8 +62,8 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
-AT_CAPTURE_FILE([pid])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_CAPTURE_FILE([pid])
 # Start the daemon and make sure that the pidfile exists immediately.
 # We don't wait for the pidfile to get created because the daemon is
 # supposed to do so before the parent exits.
@@ -78,15 +78,15 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach --monitor - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
 m4_define([CHECK], 
   [AT_CHECK([$1], [$2], [$3], [$4], [kill `cat daemon monitor`])])
-AT_CAPTURE_FILE([daemon])
-AT_CAPTURE_FILE([olddaemon])
-AT_CAPTURE_FILE([newdaemon])
-AT_CAPTURE_FILE([monitor])
-AT_CAPTURE_FILE([newmonitor])
-AT_CAPTURE_FILE([init])
+dnl AT_CAPTURE_FILE([daemon])
+dnl AT_CAPTURE_FILE([olddaemon])
+dnl AT_CAPTURE_FILE([newdaemon])
+dnl AT_CAPTURE_FILE([monitor])
+dnl AT_CAPTURE_FILE([newmonitor])
+dnl AT_CAPTURE_FILE([init])
 # Start the daemon and make sure that the pidfile exists immediately.
 # We don't wait for the pidfile to get created because the daemon is
 # supposed to do so before the parent exits.
@@ -127,8 +127,8 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach startup errors - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
-AT_CAPTURE_FILE([pid])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_CAPTURE_FILE([pid])
 AT_CHECK([$PYTHON $srcdir/test-daemon.py --pidfile-name=$PWD/pid --detach --bail], [1], [], [stderr])
 AT_CHECK([grep 'test-daemon.py: exiting after daemonize_start() as requested' stderr],
   [0], [ignore], [])
@@ -136,8 +136,8 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach --monitor startup errors - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
-AT_CAPTURE_FILE([pid])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_CAPTURE_FILE([pid])
 AT_CHECK([$PYTHON $srcdir/test-daemon.py --pidfile-name=$PWD/pid --detach --monitor --bail], [1], [], [stderr])
 AT_CHECK([grep 'test-daemon.py: exiting after daemonize_start() as requested' stderr],
   [0], [ignore], [])
@@ -145,10 +145,10 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach closes standard fds - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([status])
-AT_CAPTURE_FILE([stderr])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([status])
+dnl AT_CAPTURE_FILE([stderr])
 AT_CHECK([(yes 2>stderr; echo $? > status) | $PYTHON $srcdir/test-daemon.py --pidfile-name=$PWD/pid --detach], [0], [], [])
 AT_CHECK([kill `cat pid`])
 AT_CHECK([test -s status])
@@ -165,9 +165,9 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach --monitor closes standard fds])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([status])
-AT_CAPTURE_FILE([stderr])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([status])
+dnl AT_CAPTURE_FILE([stderr])
 OVSDB_INIT([db])
 AT_CHECK([(yes 2>stderr; echo $? > status) | $PYTHON $srcdir/test-daemon.py --pidfile-name=$PWD/pid --detach], [0], [], [])
 AT_CHECK([kill `cat pid`])
diff -r cfea8989bb9d tests/daemon.at
--- a/tests/daemon.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/daemon.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,10 +1,10 @@ AT_BANNER([daemon unit tests - C])
 AT_BANNER([daemon unit tests - C])
 
 AT_SETUP([daemon])
-AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
+dnl AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
 OVSDB_INIT([db])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([expected])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([expected])
 # Start the daemon and wait for the pidfile to get created
 # and that its contents are the correct pid.
 AT_CHECK([ovsdb-server --pidfile=$PWD/pid --remote=punix:socket --unixctl=$PWD/unixctl db& echo $! > expected], [0], [ignore], [ignore])
@@ -20,12 +20,12 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --monitor])
-AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
+dnl AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
 OVSDB_INIT([db])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([parent])
-AT_CAPTURE_FILE([parentpid])
-AT_CAPTURE_FILE([newpid])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([parent])
+dnl AT_CAPTURE_FILE([parentpid])
+dnl AT_CAPTURE_FILE([newpid])
 # Start the daemon and wait for the pidfile to get created.
 AT_CHECK([ovsdb-server --monitor --pidfile=$PWD/pid --remote=punix:socket --unixctl=$PWD/unixctl db& echo $! > parent], [0], [ignore], [ignore])
 OVS_WAIT_UNTIL([test -s pid], [kill `cat parent`])
@@ -64,8 +64,8 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach])
-AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
-AT_CAPTURE_FILE([pid])
+dnl AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
+dnl AT_CAPTURE_FILE([pid])
 OVSDB_INIT([db])
 # Start the daemon and make sure that the pidfile exists immediately.
 # We don't wait for the pidfile to get created because the daemon is
@@ -81,16 +81,16 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach --monitor])
-AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
+dnl AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov wrapper make pids differ
 m4_define([CHECK], 
   [AT_CHECK([$1], [$2], [$3], [$4], [kill `cat daemon monitor`])])
 OVSDB_INIT([db])
-AT_CAPTURE_FILE([daemon])
-AT_CAPTURE_FILE([olddaemon])
-AT_CAPTURE_FILE([newdaemon])
-AT_CAPTURE_FILE([monitor])
-AT_CAPTURE_FILE([newmonitor])
-AT_CAPTURE_FILE([init])
+dnl AT_CAPTURE_FILE([daemon])
+dnl AT_CAPTURE_FILE([olddaemon])
+dnl AT_CAPTURE_FILE([newdaemon])
+dnl AT_CAPTURE_FILE([monitor])
+dnl AT_CAPTURE_FILE([newmonitor])
+dnl AT_CAPTURE_FILE([init])
 # Start the daemon and make sure that the pidfile exists immediately.
 # We don't wait for the pidfile to get created because the daemon is
 # supposed to do so before the parent exits.
@@ -131,7 +131,7 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach startup errors])
-AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([pid])
 OVSDB_INIT([db])
 AT_CHECK([ovsdb-server --detach --pidfile=$PWD/pid --unixctl=$PWD/nonexistent/unixctl db], [1], [], [stderr])
 AT_CHECK([grep 'ovsdb-server: could not initialize control socket' stderr],
@@ -140,7 +140,7 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach --monitor startup errors])
-AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([pid])
 OVSDB_INIT([db])
 AT_CHECK([ovsdb-server --detach --pidfile=$PWD/pid --monitor --unixctl=$PWD/nonexistent/unixctl db], [1], [], [stderr])
 AT_CHECK([grep 'ovsdb-server: could not initialize control socket' stderr],
@@ -149,9 +149,9 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach closes standard fds])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([status])
-AT_CAPTURE_FILE([stderr])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([status])
+dnl AT_CAPTURE_FILE([stderr])
 OVSDB_INIT([db])
 AT_CHECK([(yes 2>stderr; echo $? > status) | ovsdb-server --detach --pidfile=$PWD/pid --unixctl=$PWD/unixctl db], [0], [], [])
 AT_CHECK([kill `cat pid`])
@@ -169,9 +169,9 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([daemon --detach --monitor closes standard fds])
-AT_CAPTURE_FILE([pid])
-AT_CAPTURE_FILE([status])
-AT_CAPTURE_FILE([stderr])
+dnl AT_CAPTURE_FILE([pid])
+dnl AT_CAPTURE_FILE([status])
+dnl AT_CAPTURE_FILE([stderr])
 OVSDB_INIT([db])
 AT_CHECK([(yes 2>stderr; echo $? > status) | ovsdb-server --detach --monitor --pidfile=$PWD/pid --unixctl=$PWD/unixctl db], [0], [], [])
 AT_CHECK([kill `cat pid`])
diff -r cfea8989bb9d tests/json.at
--- a/tests/json.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/json.at	Mon Sep 13 14:57:41 2010 +0100
@@ -2,7 +2,7 @@ m4_define([JSON_CHECK_POSITIVE_C],
   [AT_SETUP([$1])
    AT_KEYWORDS([json positive])
    AT_CHECK([printf %s "AS_ESCAPE([$2])" > input])
-   AT_CAPTURE_FILE([input])
+   dnl AT_CAPTURE_FILE([input])
    AT_CHECK([test-json $4 input], [0], [stdout], [])
    AT_CHECK([cat stdout], [0], [$3
 ])
@@ -11,9 +11,9 @@ m4_define([JSON_CHECK_POSITIVE_PY],
 m4_define([JSON_CHECK_POSITIVE_PY], 
   [AT_SETUP([$1])
    AT_KEYWORDS([json positive Python])
-   AT_SKIP_IF([test $HAVE_PYTHON = no])
+   dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
    AT_CHECK([printf %s "AS_ESCAPE([$2])" > input])
-   AT_CAPTURE_FILE([input])
+   dnl AT_CAPTURE_FILE([input])
    AT_CHECK([$PYTHON $srcdir/test-json.py $4 input], [0], [stdout], [])
    AT_CHECK([cat stdout], [0], [$3
 ])
@@ -27,7 +27,7 @@ m4_define([JSON_CHECK_NEGATIVE_C],
   [AT_SETUP([$1])
    AT_KEYWORDS([json negative])
    AT_CHECK([printf %s "AS_ESCAPE([$2])" > input])
-   AT_CAPTURE_FILE([input])
+   dnl AT_CAPTURE_FILE([input])
    AT_CHECK([test-json $4 input], [1], [stdout], [])
    AT_CHECK([[sed 's/^error: [^:]*:/error:/' < stdout]], [0], [$3
 ])
@@ -36,9 +36,9 @@ m4_define([JSON_CHECK_NEGATIVE_PY],
 m4_define([JSON_CHECK_NEGATIVE_PY], 
   [AT_SETUP([$1])
    AT_KEYWORDS([json negative Python])
-   AT_SKIP_IF([test $HAVE_PYTHON = no])
+   dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
    AT_CHECK([printf %s "AS_ESCAPE([$2])" > input])
-   AT_CAPTURE_FILE([input])
+   dnl AT_CAPTURE_FILE([input])
    AT_CHECK([$PYTHON $srcdir/test-json.py $4 input], [1], [stdout], [])
    AT_CHECK([[sed 's/^error: [^:]*:/error:/' < stdout]], [0], [$3
 ])
@@ -115,7 +115,7 @@ AT_CLEANUP
 
 AT_SETUP([end of input in quoted string - Python])
 AT_KEYWORDS([json negative Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
 AT_CHECK([printf '"xxx' > input
 $PYTHON $srcdir/test-json.py input], [1],
   [error: line 0, column 4, byte 4: unexpected end of input in quoted string
diff -r cfea8989bb9d tests/jsonrpc-py.at
--- a/tests/jsonrpc-py.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/jsonrpc-py.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,7 +1,7 @@ AT_BANNER([JSON-RPC - Python])
 AT_BANNER([JSON-RPC - Python])
 
 AT_SETUP([JSON-RPC request and successful reply - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
 AT_CHECK([$PYTHON $srcdir/test-jsonrpc.py --detach --pidfile-name=$PWD/pid listen punix:socket])
 AT_CHECK([test -s pid])
 AT_CHECK([kill -0 `cat pid`])
@@ -13,7 +13,7 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([JSON-RPC request and error reply - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
 AT_CHECK([$PYTHON $srcdir/test-jsonrpc.py --detach --pidfile-name=$PWD/pid listen punix:socket])
 AT_CHECK([test -s pid])
 AT_CHECK([kill -0 `cat pid`])
@@ -25,7 +25,7 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([JSON-RPC notification - Python])
-AT_SKIP_IF([test $HAVE_PYTHON = no])
+dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
 AT_CHECK([$PYTHON $srcdir/test-jsonrpc.py --detach --pidfile-name=$PWD/pid listen punix:socket])
 AT_CHECK([test -s pid])
 # When a daemon dies it deletes its pidfile, so make a copy.
diff -r cfea8989bb9d tests/lockfile.at
--- a/tests/lockfile.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/lockfile.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,12 +1,12 @@ AT_BANNER([lockfile unit tests])
 AT_BANNER([lockfile unit tests])
 
-m4_define([CHECK_LOCKFILE],
-  [AT_SETUP([m4_translit([$1], [_], [ ])])
-   AT_KEYWORDS([lockfile])
-   AT_CHECK([test-lockfile $1], [0], [$1: success (m4_if(
-     [$2], [1], [$2 child], [$2 children]))
-])
-   AT_CLEANUP])
+dnl m4_define([CHECK_LOCKFILE],
+dnl   [AT_SETUP([m4_translit([$1], [_], [ ])])
+dnl    AT_KEYWORDS([lockfile])
+dnl    AT_CHECK([test-lockfile $1], [0], [$1: success (m4_if(
+dnl      [$2], [1], [$2 child], [$2 children]))
+dnl ])
+dnl    AT_CLEANUP])
 
 CHECK_LOCKFILE([lock_and_unlock], [0])
 CHECK_LOCKFILE([lock_and_unlock_twice], [0])
diff -r cfea8989bb9d tests/ovsdb-idl-py.at
--- a/tests/ovsdb-idl-py.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/ovsdb-idl-py.at	Mon Sep 13 14:57:41 2010 +0100
@@ -18,7 +18,7 @@ AT_BANNER([OVSDB -- interface descriptio
 # TITLE is provided to AT_SETUP and KEYWORDS to AT_KEYWORDS.
 m4_define([OVSDB_CHECK_IDL_PY], 
   [AT_SETUP([$1])
-   AT_SKIP_IF([test $HAVE_PYTHON = no])
+   dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
    AT_KEYWORDS([ovsdb server idl positive Python $5])
    AT_CHECK([ovsdb-tool create db $abs_srcdir/idltest.ovsschema],
                   [0], [stdout], [ignore])
diff -r cfea8989bb9d tests/ovsdb-log.at
--- a/tests/ovsdb-log.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/ovsdb-log.at	Mon Sep 13 14:57:41 2010 +0100
@@ -2,7 +2,7 @@ AT_BANNER([OVSDB -- logging])
 
 AT_SETUP([create empty, reread])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([log])
+dnl AT_CAPTURE_FILE([log])
 AT_CHECK(
   [test-ovsdb log-io file create], [0], 
   [file: open successful
@@ -17,7 +17,7 @@ AT_CLEANUP
 
 AT_SETUP([write one, reread])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]']], [0], 
   [[file: open successful
@@ -34,7 +34,7 @@ AT_CLEANUP
 
 AT_SETUP([check that create fails if file exists])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[1]']], [0], 
   [[file: open successful
@@ -54,7 +54,7 @@ AT_CLEANUP
 
 AT_SETUP([write one, reread])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -75,7 +75,7 @@ AT_CLEANUP
 
 AT_SETUP([write one, reread, append])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -105,7 +105,7 @@ AT_CLEANUP
 
 AT_SETUP([write, reread one, overwrite])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -131,7 +131,7 @@ AT_CLEANUP
 
 AT_SETUP([write, add corrupted data, read])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -153,7 +153,7 @@ AT_CLEANUP
 
 AT_SETUP([write, add corrupted data, read, overwrite])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -185,7 +185,7 @@ AT_CLEANUP
 
 AT_SETUP([write, corrupt some data, read, overwrite])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -218,7 +218,7 @@ AT_CLEANUP
 
 AT_SETUP([write, truncate file, read, overwrite])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
@@ -251,7 +251,7 @@ AT_CLEANUP
 
 AT_SETUP([write bad JSON, read, overwrite])
 AT_KEYWORDS([ovsdb log])
-AT_CAPTURE_FILE([file])
+dnl AT_CAPTURE_FILE([file])
 AT_CHECK(
   [[test-ovsdb log-io file create 'write:[0]' 'write:[1]' 'write:[2]']], [0], 
   [[file: open successful
diff -r cfea8989bb9d tests/ovsdb-monitor.at
--- a/tests/ovsdb-monitor.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/ovsdb-monitor.at	Mon Sep 13 14:57:41 2010 +0100
@@ -24,7 +24,7 @@ m4_define([OVSDB_CHECK_MONITOR],
    AT_CHECK([ovsdb-tool create db schema], [0], [stdout], [ignore])
    m4_foreach([txn], [$3],
      [AT_CHECK([ovsdb-tool transact db 'txn'], [0], [ignore], [ignore])])
-   AT_CAPTURE_FILE([ovsdb-server-log])
+   dnl AT_CAPTURE_FILE([ovsdb-server-log])
    AT_CHECK([ovsdb-server --detach --pidfile=$PWD/server-pid --remote=punix:socket --unixctl=$PWD/unixctl --log-file=$PWD/ovsdb-server-log db >/dev/null 2>&1],
             [0], [], [])
    AT_CHECK([ovsdb-client -vjsonrpc --detach --pidfile=$PWD/client-pid -d json monitor --format=csv unix:socket $4 $5 $8 > output], 
diff -r cfea8989bb9d tests/ovsdb-server.at
--- a/tests/ovsdb-server.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/ovsdb-server.at	Mon Sep 13 14:57:41 2010 +0100
@@ -92,11 +92,11 @@ AT_CLEANUP
 
 AT_SETUP([SSL db: implementation])
 AT_KEYWORDS([ovsdb server positive ssl $5])
-AT_SKIP_IF([test "$HAVE_OPENSSL" = no])
-AT_SKIP_IF([test "x$RANDOM" = x])
+dnl AT_SKIP_IF([test "$HAVE_OPENSSL" = no])
+dnl AT_SKIP_IF([test "x$RANDOM" = x])
 SSL_PORT=`expr 32767 + \( $RANDOM % 32767 \)`
 PKIDIR=$abs_top_srcdir/tests
-AT_SKIP_IF([expr "$PKIDIR" : ".*[ 	'\"
+dnl AT_SKIP_IF([expr "$PKIDIR" : ".*[ 	'\"
 \\]"])
 AT_DATA([schema],
   [[{"name": "mydb",
@@ -137,10 +137,10 @@ AT_CHECK(
            "columns": ["private_key"]}]']], 
   [0], [stdout], [ignore], [test ! -e pid || kill `cat pid`])
 cat stdout >> output
-AT_CHECK_UNQUOTED(
-  [perl $srcdir/uuidfilt.pl output], [0], 
-  [[[{"rows":[{"private_key":"$PKIDIR/testpki-privkey2.pem"}]}]
-]], [ignore], [test ! -e pid || kill `cat pid`])
+dnl AT_CHECK_UNQUOTED(
+dnl   [perl $srcdir/uuidfilt.pl output], [0], 
+dnl   [[[{"rows":[{"private_key":"$PKIDIR/testpki-privkey2.pem"}]}]
+dnl ]], [ignore], [test ! -e pid || kill `cat pid`])
 OVSDB_SERVER_SHUTDOWN
 AT_CLEANUP
 
@@ -151,7 +151,7 @@ touch .db.~lock~
 touch .db.~lock~
 AT_CHECK([ovsdb-tool create db schema], [0], [ignore], [ignore])
 AT_CHECK([ovsdb-server --detach --pidfile=$PWD/pid --unixctl=$PWD/unixctl --remote=punix:socket --log-file=$PWD/ovsdb-server.log db], [0], [ignore], [ignore])
-AT_CAPTURE_FILE([ovsdb-server.log])
+dnl AT_CAPTURE_FILE([ovsdb-server.log])
 dnl Do a bunch of random transactions that put crap in the database log.
 AT_CHECK(
   [[for pair in 'zero 0' 'one 1' 'two 2' 'three 3' 'four 4' 'five 5'; do
@@ -220,7 +220,7 @@ dnl We can't fully re-check the contents
 dnl We can't fully re-check the contents of the database log, because the
 dnl order of the records is not predictable, but there should only be 4 lines
 dnl in it now.
-AT_CAPTURE_FILE([db])
+dnl AT_CAPTURE_FILE([db])
 AT_CHECK([wc -l < db], [0], [4
 ], [], [test ! -e pid || kill `cat pid`])
 dnl And check that the dumped data is the same too:
@@ -280,8 +280,8 @@ m4_define([OVSDB_CHECK_EXECUTION],
 m4_define([OVSDB_CHECK_EXECUTION], 
   [AT_SETUP([$1])
    AT_KEYWORDS([ovsdb server positive ssl $5])
-   AT_SKIP_IF([test "$HAVE_OPENSSL" = no])
-   AT_SKIP_IF([test "x$RANDOM" = x])
+   dnl AT_SKIP_IF([test "$HAVE_OPENSSL" = no])
+   dnl AT_SKIP_IF([test "x$RANDOM" = x])
    AT_DATA([schema], [$2
 ])
    SSL_PORT=`expr 32767 + \( $RANDOM % 32767 \)`
diff -r cfea8989bb9d tests/ovsdb-tool.at
--- a/tests/ovsdb-tool.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/ovsdb-tool.at	Mon Sep 13 14:57:41 2010 +0100
@@ -122,7 +122,7 @@ dnl We can't fully re-check the contents
 dnl We can't fully re-check the contents of the database log, because the
 dnl order of the records is not predictable, but there should only be 4 lines
 dnl in it now.
-AT_CAPTURE_FILE([db])
+dnl AT_CAPTURE_FILE([db])
 AT_CHECK([wc -l < db], [0], [4
 ])
 dnl And check that the dumped data is the same too:
@@ -185,7 +185,7 @@ dnl We can't fully re-check the contents
 dnl We can't fully re-check the contents of the database log, because the
 dnl order of the records is not predictable, but there should only be 4 lines
 dnl in it now.
-AT_CAPTURE_FILE([db])
+dnl AT_CAPTURE_FILE([db])
 AT_CHECK([wc -l < db], [0], [4
 ])
 dnl And check that the dumped data is the same except for the removed column:
@@ -247,7 +247,7 @@ dnl We can't fully re-check the contents
 dnl We can't fully re-check the contents of the database log, because the
 dnl order of the records is not predictable, but there should only be 4 lines
 dnl in it now.
-AT_CAPTURE_FILE([db])
+dnl AT_CAPTURE_FILE([db])
 AT_CHECK([wc -l < db], [0], [4
 ])
 dnl And check that the dumped data is the same except for the added column:
diff -r cfea8989bb9d tests/ovsdb.at
--- a/tests/ovsdb.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/ovsdb.at	Mon Sep 13 14:57:41 2010 +0100
@@ -23,7 +23,7 @@ m4_define([OVSDB_CHECK_POSITIVE],
 # TITLE is provided to AT_SETUP and KEYWORDS to AT_KEYWORDS.
 m4_define([OVSDB_CHECK_POSITIVE_PY], 
   [AT_SETUP([$1])
-   AT_SKIP_IF([test $HAVE_PYTHON = no])
+   dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
    m4_if([$6], [], [], [AT_XFAIL_IF([:])])
    AT_KEYWORDS([ovsdb positive Python $4])
    AT_CHECK([$PYTHON $srcdir/test-ovsdb.py $2], [0], [$3
@@ -65,7 +65,7 @@ m4_define([OVSDB_CHECK_NEGATIVE],
 # TITLE is provided to AT_SETUP and KEYWORDS to AT_KEYWORDS.  
 m4_define([OVSDB_CHECK_NEGATIVE_PY], 
   [AT_SETUP([$1])
-   AT_SKIP_IF([test $HAVE_PYTHON = no])
+   dnl AT_SKIP_IF([test $HAVE_PYTHON = no])
    AT_KEYWORDS([ovsdb negative $4])
    AT_CHECK([$PYTHON $srcdir/test-ovsdb.py $2], [1], [], [stderr])
    m4_assert(m4_len([$3]))
diff -r cfea8989bb9d tests/reconnect.at
--- a/tests/reconnect.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/reconnect.at	Mon Sep 13 14:57:41 2010 +0100
@@ -17,7 +17,7 @@ m4_define([RECONNECT_CHECK],
      [$3])
    __RECONNECT_CHECK(
      [$1 - Python],
-     [AT_SKIP_IF([test $HAVE_PYTHON = no])],
+     [dnl AT_SKIP_IF([test $HAVE_PYTHON = no])],
      [$2],
      [$PYTHON $srcdir/test-reconnect.py < input],
      [$3])])
diff -r cfea8989bb9d tests/uuid.at
--- a/tests/uuid.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/uuid.at	Mon Sep 13 14:57:41 2010 +0100
@@ -32,7 +32,7 @@ AT_CLEANUP
 AT_CLEANUP
 
 AT_SETUP([UUID generation, parsing, serialization])
-AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov makes this test absurdly slow
+dnl AT_SKIP_IF([test "$CHECK_LCOV" = true]) # lcov makes this test absurdly slow
 AT_KEYWORDS([UUID])
 AT_CHECK([
   uuids=
diff -r cfea8989bb9d tests/vconn.at
--- a/tests/vconn.at	Mon Sep 13 11:10:19 2010 +0100
+++ b/tests/vconn.at	Mon Sep 13 14:57:41 2010 +0100
@@ -1,21 +1,21 @@ m4_define([TEST_VCONN_CLASS],
-m4_define([TEST_VCONN_CLASS],
-  [AT_BANNER([vconn library -- $1 class])
-   m4_foreach(
-     [testname], 
-     [[refuse-connection], 
-      [accept-then-close],
-      [read-hello],
-      [send-plain-hello],
-      [send-long-hello],
-      [send-echo-hello],
-      [send-short-hello],
-      [send-invalid-version-hello]],
-     [AT_SETUP([$1 vconn - m4_bpatsubst(testname, [-], [ ])])
-      m4_if([$1], [ssl], [
-        AT_SKIP_IF([test "$HAVE_OPENSSL" = no])
-        AT_CHECK([cp $abs_top_srcdir/tests/testpki*.pem .])])
-      AT_CHECK([test-vconn testname $1], [0], [], [ignore])
-      AT_CLEANUP])])
+dnl m4_define([TEST_VCONN_CLASS],
+dnl   [AT_BANNER([vconn library -- $1 class])
+dnl    m4_foreach(
+dnl      [testname], 
+dnl      [[refuse-connection], 
+dnl       [accept-then-close],
+dnl       [read-hello],
+dnl       [send-plain-hello],
+dnl       [send-long-hello],
+dnl       [send-echo-hello],
+dnl       [send-short-hello],
+dnl       [send-invalid-version-hello]],
+dnl      [AT_SETUP([$1 vconn - m4_bpatsubst(testname, [-], [ ])])
+dnl       m4_if([$1], [ssl], [
+dnl         dnl AT_SKIP_IF([test "$HAVE_OPENSSL" = no])
+dnl         AT_CHECK([cp $abs_top_srcdir/tests/testpki*.pem .])])
+dnl       AT_CHECK([test-vconn testname $1], [0], [], [ignore])
+dnl       AT_CLEANUP])])
 
 TEST_VCONN_CLASS([unix])
 TEST_VCONN_CLASS([tcp])
